var/spack/repos/builtin/packages/rocm-device-libs/package.py

```diff
@@ -11,7 +11,7 @@ class RocmDeviceLibs(CMakePackage):
     """set of AMD specific device-side language runtime libraries"""
 
     homepage = "https://github.com/ROCm/ROCm-Device-Libs"
-    git = "ssh://<user-id>@gerrit-git.amd.com:<port-number>/lightning/ec/device-libs.git"
+    git = "ssh://gerritgit/lightning/ec/device-libs.git"
```

Comment:
rocm-device-libs is now part of the llvm-project. perhaps this and comgr needs to be pointing to gerritgit/lightning/ec/llvm-project.git.

Reply:
Can this one be address in a different PR?

Reply:
Yes. We need to update the branch name for llvm-amdgpu as well. We can update the URL as well with that changes.